docs/sources/mimir/references/glossary.md

```diff
@@ -34,14 +34,14 @@ Grafana Mimir comprises several components.
 Each component provides a specific function to the system.
 For component specific documentation, refer to one of the following topics:
 
-- [Compactor]({{< relref "../operators-guide/architecture/components/compactor/index.md" >}})
-- [Distributor]({{< relref "../operators-guide/architecture/components/distributor.md" >}})
-- [Ingester]({{< relref "../operators-guide/architecture/components/ingester.md" >}})
-- [Query-frontend]({{< relref "../operators-guide/architecture/components/query-frontend/index.md" >}})
-- [Query-scheduler]({{< relref "../operators-guide/architecture/components/query-scheduler/index.md" >}})
-- [Store-gateway]({{< relref "../operators-guide/architecture/components/store-gateway.md" >}})
-- [Optional: Alertmanager]({{< relref "../operators-guide/architecture/components/alertmanager.md" >}})
-- [Optional: Ruler]({{< relref "../operators-guide/architecture/components/ruler/index.md" >}})
+- [Compactor]({{< relref "architecture/components/compactor/index.md" >}})
```

Comment:
These relrefs can resolve ambiguously because they don't have a preceding `./`.

More recent versions of `doc-validator` will error on this.

You can check locally with `make update doc-validator`.